framework/packages/abstract-client/src/account.rs

```diff
@@ -382,6 +382,31 @@ impl<Chain: CwEnv> Account<Chain> {
             )
             .map_err(Into::into)
     }
+    /// Executes a [`CosmosMsg`] on the module of the account.
+    pub fn execute_on_module<M>(
+        &self,
+        execute_msg: &M::ExecuteMsg,
+        funds: &[Coin],
+    ) -> AbstractClientResult<<Chain as TxHandler>::Response>
+    where
+        M: ContractInstance<Chain>
+            + ExecutableContract
+            + DependencyCreation
+            + InstallConfig
```

Comment:
Most of these trait bounds can be removed afaik

Reply:
Yes !